FirebaseAuth/Sources/Swift/User/User.swift

```diff
@@ -1369,10 +1369,21 @@ extension User: NSSecureCoding {}
    */
   var tokenService: SecureTokenService
 
+  private weak var _auth: Auth?
   /** @property auth
       @brief A weak reference to a FIRAuth instance associated with this instance.
    */
-  weak var auth: Auth?
+  weak var auth: Auth? {
```

Comment:
This was missed in initial implementation and by unit tests.  See the comment above. This code is required to initialize requestConfiguration when the User and SecureTokenService are decoded on startup from the keychain.